release.md

```diff
@@ -65,28 +77,31 @@ Also define these variables for the versions you are releasing:
 
 ```bash
 VER=VERSION_YOU_WANT_TO_RELEASE  # e.g. "0.7.0"
-NEXT_VER=NEXT_VERSION  # e.g. "0.8.0"
+NEXT_VER=NEXT_VERSION  # e.g. "0.8.0" (skip for PATCH releases)
 ```
 
 ### Create release branch
 
 Create a release branch called "v${VERSION}-dev":
 
 ```bash
-git checkout master -b "v${VER}-dev"
-vi _version.py   # Remove .dev from version
-git add _version.py
-git commit -m "bla bla"
+git checkout -b "v${VER}-dev"
+vi ./cirq/_version.py   # Remove .dev from version
+git add ./cirq/_version.py
```

Comment:
I don't think you need to add it, since the file already exists, correct?

Reply:
`git add` is required even if the file is already tracked by git, this adds select modifications to the index, so when using `git commit -m "message"`, only the `git add`-ed changes are committed. It's a conservative approach compared to `git commit -am "message"` which is equivalent to "add modifications to _all_ changed tracked files to the index" - which could result accidentally committing other files if you accidentally changed them in the repo.